beesdoo_website_shift/views/my_shift_website_templates.xml

```diff
@@ -380,7 +380,7 @@
       <t t-set="shift" t-value="shift_count_subscribed[0]" />
       <t t-set="count" t-value="shift_count_subscribed[1]" />
       <t t-set="is_subscribed" t-value="shift_count_subscribed[2]" />
-      <t t-set="highlight_class" t-value="'panel-warning' if count >= highlight_rule else 'panel-default'"/>
+      <t t-set="highlight_class" t-value="'panel-warning' if not has_enough_workers else 'panel-default'"/>
```

Comment:
Here you cannot access to `has_enough_workers` directly. This variable works like `shift` or `count`. Look at these variable to see how it works. And be sure to have a good understanding of what's going on with these variable. If you have a doubt, ask me. :)

Reply:
Ok, it's fixed.